docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -264,13 +264,18 @@ Then install Ansible with [1]_::
 
 Or if you are looking for the latest development version::
 
-    pip install git+https://github.com/ansible/ansible.git@devel
+   $ pip install git+https://github.com/ansible/ansible.git@devel
 
 If you are installing on OS X Mavericks, you may encounter some noise from your compiler.  A workaround is to do the following::
 
    $ sudo CFLAGS=-Qunused-arguments CPPFLAGS=-Qunused-arguments pip install ansible
 
-Readers that use virtualenv can also install Ansible under virtualenv, though we'd recommend to not worry about it and just install Ansible globally.  Do not use easy_install to install ansible directly.
+Readers that use virtualenv can also install Ansible under virtualenv, though we'd recommend to not worry about it and just install Ansible globally.  Do not use easy_install to install Ansible directly.
+
+.. note::
+
+    Older versions of pip defaults to http://pypi.python.org/simple, which no longer works.
```

Comment:
I think, it's appropriate to refer to the `get-pip.py` doc here.
Also I think it's v9.3 + it's worth mentioning that it comes with newer TLS support bundled, which is crucial for MacOS users: https://mail.python.org/pipermail/python-announce-list/2018-April/011885.html